src/markdown-pages/create-installer/index.md

```diff
@@ -30,7 +30,13 @@ spec:
 ```
 
 ## Versions
-For add-ons that are using `version: "latest"` this will be pinned to the latest version of the component that is supported by our installer. This means that when an update to the component is shipped, your installer will automatically be updated. This may be desirable in some scenarios, while other installers may want to have tested, locked and predictable installed versions. You can also list a specific (supported) version of an add on and it will be locked to that version.
+You can specify a (supported) version of an add-on and it will be locked to that version.
```

Comment:
You may want to add a little more color here

Reply:
This is why I ask you to review. No perfunctory reviews.